storage/innobase/dict/dict0stats.cc

```diff
@@ -1390,27 +1468,25 @@ dict_stats_analyze_index()
       dict_stats_analyze_index_below_cur()
 @} */
 
-/*********************************************************************//**
-Find the total number and the number of distinct keys on a given level in
-an index. Each of the 1..n_uniq prefixes are looked up and the results are
-saved in the array n_diff[0] .. n_diff[n_uniq - 1]. The total number of
-records on the level is saved in total_recs.
-Also, the index of the last record in each group of equal records is saved
-in n_diff_boundaries[0..n_uniq - 1], records indexing starts from the leftmost
-record on the level and continues cross pages boundaries, counting from 0. */
+/** Find the total number and the number of distinct keys on a given
+level in an index. Each of the 1..n_uniq prefixes are looked up and
+the results are saved in the array n_diff[0] .. n_diff[n_uniq - 1].
+The total number of records on the level is saved in total_recs.
+Also, the index of the last record in each group of equal records is
+saved in n_diff_boundaries[0..n_uniq - 1], records indexing starts
+from the leftmost record on the level and continues cross pages
+boundaries, counting from 0.
+@param index index
+@param level level for the index
+@param stats statistics extracted from index
+@param mtr   mini-transaction */
```

Comment:
The parameter documentation does not help much. Let’s give it some better attempt:
```
@param index  b-tree
@param level  distance to analyze from the from leaf page level (0)
```